source/extensions/all_extensions.bzl

```diff
@@ -28,3 +29,9 @@ def envoy_windows_extensions():
         windows_extensions.append(path)
 
     return windows_extensions
+
+def is_blacklisted_extension(extension, blacklist):
+    for be in blacklist:
+        if be in extension:
```

Comment:
This tests for substring search, is this really what you want? Do you want equality, suffix match, etc.? Seems too weak as is.

Reply:
I agree. Not the best. I was worried about potential shared properties in names, e.g. `envoy.a.x` and `envoy.a.y`, where maybe x would be really common, like `net`, but really, you wanted to filter out `a.x`. In that case, I was envisioning that you'd have to add `a.x` to your blacklist (though that would break the name match from `TARGET_RECIPES`, but I think you could just rename whatever it is in TARGET_RECIPES if you need to go that route?).

I could only match the last token:
```
def is_blacklisted_extension(extension, blacklist):                      
    for be in blacklist:
        if extension.split(".")[-1] == be:
                return True 
    return False
```

Reply:
Can you just make it an exact match?

Reply:
Sure, doesn't seem too bad.

Reply:
sorry -- got sucked into 10 new things this week. 

If we did that instead of the shorter name, it would mess up `_build_recipe_repository_impl`,  which uses the short name for the recipe script match. 
Another option is to do that in addition to the other ones. so the list would have two-three different required matches for everything we want to skip, e.g. 
`PPC_SKIP_TARGETS=["lua", "luajit", "envoy.filters.http.lua"]` but then I think we still have the potential to run into the original problem I was worried about, since the shorter ones need to stay.

Reply:
Actually, since the shorter ones wouldn't match in this usage of that list, I think that would be fine. But gross, with all those different representations of the same thing.

Reply:
ah, nice. i redid with a dict. in one place i check keys. in another i check values. thanks for that! lmk what you think @htuch